runtime/codert_vm/decomp.cpp

```diff
@@ -352,15 +364,38 @@ jitCleanUpDecompilationStack(J9VMThread * currentThread, J9StackWalkState * walk
 #endif /* J9VM_INTERP_HOT_CODE_REPLACEMENT (autogen) */
 
 
+#if JAVA_SPEC_VERSION >= 19
+static jvmtiIterationControl
+codeBreakpointAddedCallBack(J9VMThread *currentThread, J9MM_IterateObjectDescriptor *object, void *userData)
+{
+	J9JavaVM *vm = currentThread->javaVM;
+	j9object_t continuationObj = object->object;
+	j9object_t vthread = J9VMJDKINTERNALVMCONTINUATION_VTHREAD(currentThread, continuationObj);
+	ContinuationState continuationState = J9VMJDKINTERNALVMCONTINUATION_STATE(currentThread, continuationObj);
+
+	if ((NULL != vthread) && J9_ARE_NO_BITS_SET(continuationState, J9_GC_CONTINUATION_STATE_LAST_UNMOUNT)) {
```

Comment:
This check might be redundant. `walkContinuationStackFrames` has a check for `(NULL != continuation)`, which will cover both of these scenarios. @fengxue-IS Can you confirm?

Reply:
I copied this from `jvmtiSuspendResumeCallBack`. There's also a case in here that doesn't walk the stack.

Reply:
After looking at the old commits, this check seems safer than `(NULL != continuation)`.